### PR TITLE
devex-1180 - pulling in rosies work on split tests

### PR DESCRIPTION
**Pull request by codyseibert**

### CODE REVIEW

**Review by rgoshen — APPROVED** [review]

File: .github/workflows/client.yml [hunk 1]
```diff
@@ -19,4 +26,30 @@ jobs:
...
+    runs-on: ubuntu-latest
+    needs: [Client]
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/download-artifact@v3
+        with:
+          name: coverage-artifacts
+          path: coverage
+      - name: Process Coverage
+        run: npx nyc report --check-coverage --branches 94.56 --functions 98.81 --lines 99.25 --statements 99.23 --reporter lcov --reporter text --reporter clover -t coverage
```
Comment (rosiehsmith) [inline .github/workflows/client.yml:51]: sweet!

File: split-tests-glob.ts [hunk 2]
```diff
@@ -0,0 +1,16 @@
+import glob from 'glob';
+import shuffleSeed from 'shuffle-seed';
+
+const testType = process.argv[2] || '';
+
+let testFiles;
+if (testType.includes('unit')) {
+  // could use for shared? pass the path?
```
Comment (rosiehsmith) [inline split-tests-glob.ts]: could cleanup this comment i had added

**Review by rosiehsmith — APPROVED** [review]

File: web-client/jest-unit.config.ts [hunk 3]
```diff
@@ -1,4 +1,7 @@
 import type { Config } from 'jest';
+const testFilesEnv = process.env.TESTFILES;
+const testFiles = testFilesEnv?.split(' ') || [];
+console.log('testFiles in config', testFiles);
```
Comment (rosiehsmith) [inline web-client/jest-unit.config.ts]: woops one more log

**Review by mmarcotte — APPROVED** [review]
👏 👏 👏 🙏

**Pull request merged**